includes/html/pages/device/routing/bgp.inc.php

```diff
@@ -121,6 +121,9 @@
     unset($alert);
     unset($peerhost, $peername);
 
+    // load the peer identifier into an object
+    $peer['bgpPeerIdentifierObject'] = IP::parse($peer['bgpPeerIdentifier'], true);
```

Comment:
Why are we setting this in the $peer array and not as a regular variable?

Reply:
It's a mutation of an existing value of the array, so my preference is to keep it within the array.

Happy to change it to an individual variable.

Reply:
It's fine, I was worried about making future code changes difficult, but looking at this, it needs to be thrown away and created fresh when ported to Laravel anyway...